services/api/service.go

```diff
@@ -660,9 +660,9 @@ func (api *RelayAPI) RespondError(w http.ResponseWriter, code int, message strin
 	}
 }
 
-func (api *RelayAPI) RespondOK(w http.ResponseWriter, response any) {
+func (api *RelayAPI) RespondOK(w http.ResponseWriter, code int, response any) {
```

Comment:
Might as well rename to "respond" now. Maybe add another method?

Reply:
renamed respond! i like one function :-) the error one is difference because it takes a string specifically.